app/javascript/app/components/ndc-sdg/ndc-sdg-linkages-map/ndc-sdg-linkages-map-component.jsx

```diff
@@ -60,31 +61,28 @@ class NdcSdgLinkagesMap extends PureComponent {
     this.setState({ country: country.properties });
 
   renderMapHeader({ isTablet }) {
-    const { downloadLink } = this.props;
+    const { downloadLink, handleInfoClick } = this.props;
     const buttonGroup = (
-      <ButtonGroup
-        className={styles.buttons}
-        buttonsConfig={[
-          {
-            type: 'info',
-            onClick: this.props.handleInfoClick
-          },
-          {
-            type: 'share',
-            shareUrl: '/embed/ndcs-sdg',
-            analyticsGraphName: 'Ndcs-Sdg',
-            positionRight: true
-          },
-          {
-            type: 'download',
-            section: 'ndcs-sdg',
-            link: downloadLink
-          },
-          {
-            type: 'addToUser'
-          }
-        ]}
-      />
+      <React.Fragment>
+        <ButtonGroup
+          className={styles.buttons}
+          buttonsConfig={[
+            {
+              type: 'info',
+              onClick: handleInfoClick
+            },
+            {
+              type: 'download',
+              section: 'ndcs-content',
+              link: downloadLink
+            },
+            {
+              type: 'addToUser'
+            }
+          ]}
+        />
+        <ShareButton sharePath={'/embed/ndcs-sdg'} />
```

Comment:
Remove the `/embed` part, it's being duplicated in the URL:`http://localhost:3000/embed/embed/ndcs-sdg`
```suggestion
        <ShareButton sharePath={'/ndcs-sdg'} />
```